app/src/main/java/com/monkeyteam/chimpagne/ui/account/AccountSettingsScreen.kt

```diff
@@ -0,0 +1,102 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.Logout
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.material.icons.filled.Logout
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.monkeyteam.chimpagne.R
+import com.monkeyteam.chimpagne.ui.components.ChimpagneSpacer
+import com.monkeyteam.chimpagne.ui.components.GoBackButton
+import com.monkeyteam.chimpagne.ui.components.IconTextButton
+import com.monkeyteam.chimpagne.ui.components.ProfileIcon
+import com.monkeyteam.chimpagne.ui.theme.ChimpagneFontFamily
+import com.monkeyteam.chimpagne.viewmodels.AccountViewModel
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AccountSettingsScreen(
+    accountViewModel: AccountViewModel,
+    onGoBack: () -> Unit,
+    onEditRequest: () -> Unit,
+    onLogout: () -> Unit
+) {
+  val accountViewModelState by accountViewModel.uiState.collectAsState()
+
+  Scaffold(
+      topBar = {
+        TopAppBar(
```

Comment:
I believe we should start thinking about a ChimpagneTopBar component, currently the look of the top bar is not consistent throughout the app.